packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts

```diff
@@ -0,0 +1,111 @@
+import {httpClient} from "../../../../common/http/core/http-client";
+import {HttpMethod} from "../../../../common/http/core/http-method";
+import {HttpRequest} from "../../../../common/http/core/http-request";
+import {createAction} from "../../../../framework/action/action";
+import { AuthPropertyValue, Property } from "../../../../framework/property/prop.model";
+import * as mailchimp from "@mailchimp/mailchimp_marketing";
+
+
+export const addMemberToList = createAction({
+    name: 'add_member_to_list',
+    displayName: "Add Member to List",
+    description: "Add a member to an existing Mailchimp list",
+    props: {
+        authentication: Property.OAuth2({
+            description: "",
+            displayName: 'Authentication',
+            authUrl: "https://login.mailchimp.com/oauth2/authorize",
+            tokenUrl: "https://login.mailchimp.com/oauth2/token",
```

Comment:
Please move the common properties to common folder, same for base Url.

That would make it easier to write more actions / trigger

Reply:
done